### PR TITLE
Adding features

### PR DESCRIPTION
**Pull request by PeithonKing**
File "Finding District ID.py" ha s been updated to accept districts within states as "names" and not as an ID. This makes it easier for the user to work with this program.

Now it also checks for slots directly after generating the District ID!

### CODE REVIEW

**Comment by shaileshaanand** [comment]
Please rename "Finding District ID.py" to remove all spaces and use all lowercase letters.
Spaces should not be used in python script names because they can cause a lot of issues see https://stackoverflow.com/questions/9123517/how-do-you-import-a-file-in-python-with-spaces-in-the-name#:~:text=4%20Answers&text=You%20should%20take%20the%20spaces,supported%20by%20the%20import%20statement.

**Comment by shaileshaanand** [comment]
Also please use flake8 to properly format your code.
see https://simpleisbetterthancomplex.com/packages/2016/08/05/flake8.html

File: src/check.py [hunk 1]
```diff
@@ -1,25 +1,44 @@
 import requests
 from pprint import pprint
 from datetime import date
-import simpleaudio as sa
+from playsound import playsound
+# import simpleaudio as sa
```
Comment (shaileshaanand) [inline src/check.py]: remove code comments

File: src/check.py [hunk 2]
```diff
@@ -1,25 +1,44 @@
...
 import time
 import click
 
 
 def play_alarm(count: int):
-    sound_file = 'assets/alarm.wav'
+    # sound1 = "../assets/alarm.wav"
+    sound2 = "../assets/alarmVar.wav"
+    # sound3 = "../assets/beep1.mpeg"
+    # sound4 = "../assets/beep2.mpeg"
```
Comment (shaileshaanand) [inline src/check.py]: No code comments

File: src/check.py [hunk 3]
```diff
@@ -1,25 +1,44 @@
...
+        playsound(sound2)
 
+# play_alarm(5)
+'''
 def sleep_with_progress(seconds: int):
     for _ in range(seconds):
         print(".", flush=True, end="")
         time.sleep(1)
     print()
+'''
```
Comment (shaileshaanand) [inline src/check.py]: Remove

File: src/check.py [hunk 4]
```diff
@@ -1,25 +1,44 @@
...
 def play_alarm(count: int):
-    sound_file = 'assets/alarm.wav'
+    # sound1 = "../assets/alarm.wav"
+    sound2 = "../assets/alarmVar.wav"
+    # sound3 = "../assets/beep1.mpeg"
+    # sound4 = "../assets/beep2.mpeg"    
     for _ in range(count):
-        sa.WaveObject.from_wave_file(sound_file).play().wait_done()
-
+        # sa.WaveObject.from_wave_file(sound2).play().wait_done()
```
Comment (shaileshaanand) [inline src/check.py]: remove comment

**Review by shaileshaanand — CHANGES_REQUESTED** [review]
Please make these changes

**Comment by PeithonKing** [comment]
I have made the required changes, but I had some problems..... In your code
also, there were some lines which exceeded 79 characters. So I also didn't
change the lines written by me which contained more than 79 characters.

**Pull request merged**